### PR TITLE
CR-1208 | adding OTHER estabType enum type to have code as "other"

### PR DESCRIPTION
**Pull request by akritisinhaa**
This is a bug. When an address modification occurs and the estabType sent by Serco is OTHER, we send the string code field from the common service EstabType to RM in the AddressModified event.

This is currently "" and RM do not like that. For that reason we are changing it to have "OTHER" instead.
The EstabType enum has been modified and EstabTypeTest has been updated to test this.

For more information please look at https://collaborate2.ons.gov.uk/jira/browse/CR-1208

### CODE REVIEW

File: src/main/java/uk/gov/ons/ctp/common/domain/EstabType.java [hunk 1]
```diff
@@ -29,24 +27,19 @@
   APPROVED_PREMISES("APPROVED PREMISES", AddressType.CE, SecurityType.SECURE),
   ROUGH_SLEEPER("ROUGH SLEEPER", AddressType.CE, SecurityType.NOT_SECURE),
   STAFF_ACCOMMODATION("STAFF ACCOMMODATION", AddressType.CE, SecurityType.NOT_SECURE),
-  CAMPHILL("CAMPHILL", AddressType.CE, SecurityType.NOT_SECURE),
-  HOLIDAY_PARK("HOLIDAY PARK", AddressType.CE, SecurityType.NOT_SECURE),
+
```
Comment (MartinHumphrey) [inline src/main/java/uk/gov/ons/ctp/common/domain/EstabType.java:30]: Why add the empty line? Maybe just remove it?

File: src/test/java/uk/gov/ons/ctp/common/domain/EstabTypeTest.java [hunk 2]
```diff
@@ -45,5 +45,6 @@ public void unknownTurnsIntoOther() {
     assertEquals(EstabType.OTHER, otherEstabType);
     // and that OTHER addressType is empty optional
     assert (otherEstabType.getAddressType().isEmpty());
+    assert (otherEstabType.getCode().equals("OTHER"));
```
Comment (MartinHumphrey) [inline src/test/java/uk/gov/ons/ctp/common/domain/EstabTypeTest.java:48]: Addition of this seems a bit superfluous? The previous statement _assertEquals(EstabType.OTHER, otherEstabType);_ checks the enum is of type OTHER and has all the associated properties, i.e. code is going to be "OTHER". Maybe remove as redundant code?

**Review by MartinHumphrey — CHANGES_REQUESTED** [review]
Couple of observations.

File: src/main/java/uk/gov/ons/ctp/common/domain/EstabType.java (hunk 1 above)
Comment (akritisinhaa) [reply src/main/java/uk/gov/ons/ctp/common/domain/EstabType.java]: done

File: src/test/java/uk/gov/ons/ctp/common/domain/EstabTypeTest.java (hunk 2 above)
Comment (akritisinhaa) [reply src/test/java/uk/gov/ons/ctp/common/domain/EstabTypeTest.java]: done

**Review by MartinHumphrey — APPROVED** [review]

**Review by cookel2 — COMMENTED** [review]
Some of the estab_type values in https://collaborate2.ons.gov.uk/confluence/pages/viewpage.action?spaceKey=SDC&title=SERCO+-+Estab+Type+List are missing from the enum list e.g. HOTEL, HIGH_SECURE_MENTAL_HEALTH. Do these need to be added?

**Review by cookel2 — APPROVED** [review]
looks good

**Pull request merged**